OrbitGl/GpuTrack.cpp

```diff
@@ -365,27 +179,6 @@ const TextBox* GpuTrack::GetRight(TextBox* text_box) const {
   return nullptr;
 }
 
-//-----------------------------------------------------------------------------
-const TextBox* GpuTrack::GetUp(TextBox* text_box) const {
-  const Timer& timer = text_box->GetTimer();
-  return GetFirstBeforeTime(timer.m_Start, timer.m_Depth - 1);
-}
-
-//-----------------------------------------------------------------------------
-const TextBox* GpuTrack::GetDown(TextBox* text_box) const {
-  const Timer& timer = text_box->GetTimer();
-  return GetFirstAfterTime(timer.m_Start, timer.m_Depth + 1);
-}
-
-//-----------------------------------------------------------------------------
-std::vector<std::shared_ptr<TimerChain>> GpuTrack::GetAllChains() {
-  std::vector<std::shared_ptr<TimerChain>> chains;
-  for (const auto& pair : timers_) {
-    chains.push_back(pair.second);
-  }
-  return chains;
-}
-
 //-----------------------------------------------------------------------------
 std::string GpuTrack::GetBoxTooltip(PickingID id) const {
```

Comment:
"id" is not used.

Reply:
It's used on the first line (maybe changed?).